src/test/java/it/gov/pagopa/ranking/connector/rest/pdv/PdvErrorDecoderExt.java

```diff
@@ -0,0 +1,22 @@
+package it.gov.pagopa.ranking.connector.rest.pdv;
+
+import feign.Response;
+
+public class PdvErrorDecoderExt extends PdvErrorDecoder{
```

Comment:
More than an extended version, it's a spy
```suggestion
public class PdvErrorDecoderSpy extends PdvErrorDecoder{
```